### ISSUE TITLE
Why did you not get the container?

### ISSUE THREAD

**Issue by ruter1224** [pi-node/instructions#156]
![共識容器](https://user-images.githubusercontent.com/56575523/125901197-78f4e980-3968-4df4-b041-b12f367ccd0a.jpg)
![共識容器-1](https://user-images.githubusercontent.com/56575523/125901203-8bbc4fd9-5038-40fb-a574-b4fecb9a4804.jpg)

Does anyone know why the container is not obtained?
Because of storage space issues, I upgraded to a new SSD
After reinstalling the PI node, the consensus container cannot be obtained

**Comment by onsenboys** [comment]
Hello.
Me too,same to you.
I lost the consensus container when I did resetting my PC and reinstalled Pi node....

**Comment by ihatejam** [comment]
@ruter1224 The consensus container has not been pushed down to your node yet. If all the port-forwarding configuration changes have been made, and the ports are open when you go through "Visit Tech Setup" on the Node switch page - then you just need to wait until your PC is recognized by the network as being suitable for running a node. Once that happens, the consensus container will be pushed down to your PC. It can take a while for this to happen.

**Comment by conantam** [comment]
Would the Pi system select the right Node automatically or manually?

**Comment by ihatejam** [comment]
> Would the Pi system select the right Node automatically or manually?

@conantam Can you rephrase your question? I need a bit more context to understand what you mean. 
If your issue unrelated to the one in this thread started by rutter1224, then please start a separate thread.
